src/main.js

```diff
@@ -1 +1,25 @@
-console.log('Hello, world!');
+const form = document.forms[0];
+const monthly = form.elements.plan[1];
+monthly.addEventListener('mousedown', function() {
+  document.querySelector('.form__total').textContent = '30$';
+  document.querySelector('.form__pay span').textContent = '30$';
+});
+const annually = form.elements.plan[0];
+annually.addEventListener('mousedown', function() {
+  document.querySelector('.form__total').textContent = '300$';
+  document.querySelector('.form__pay span').textContent = '300$';
+});
+
+// enable button
+document.querySelector('.form__pay').disabled = false;
+
+// feature-accordition
+
+// please check line 23 !!!!
+// need a bit help for the last part
+const question = document.querySelectorAll('.accordion__question');
```

Comment:
You need to query the parent `.accordion` component here. Notice that in the GIF that I posted the `accordion--active` class is added to the parent component not the question.

![acc](https://camo.githubusercontent.com/88e39f72c99128418c61af5168a99e243b4bc902/68747470733a2f2f7265732e636c6f7564696e6172792e636f6d2f796963662f696d6167652f75706c6f61642f775f3430302f76313536313733353736382f436f64652532305468652532305765622f6163746976652e676966)